dataland-frontend/src/components/forms/CreateSmeDataset.vue

```diff
@@ -171,21 +211,15 @@ export default defineComponent({
       documentsToUpload: [] as DocumentToUpload[],
       referencedReportsForPrefill: {} as { [key: string]: CompanyReport },
       listOfFilledKpis: [] as Array<string>,
+      reportingPeriod: undefined as undefined | Date,
     };
   },
   computed: {
-    yearOfReportingDate: {
-      get(): string {
-        const reportingDataInSmeDataset = this.companyAssociatedSmeData.data?.general?.basicInformation?.reportingDate;
-        if (reportingDataInSmeDataset === undefined) {
-          return "";
-        } else {
-          return reportingDataInSmeDataset.split("-")[0];
-        }
-      },
-      set() {
-        // IGNORED
-      },
+    reportingPeriodYear(): number {
+      if (this.reportingPeriod) {
+        return this.reportingPeriod.getFullYear();
+      }
+      return 0;
```

Comment:
is there a way to use null/undefined as default?

Reply:
Looking into this